lib/src/core/transaction_information.dart

```diff
@@ -128,5 +128,5 @@ class TransactionReceipt {
         contractAddress = map['contractAddress'] != null
             ? EthereumAddress.fromHex(map['contractAddress'] as String)
             : null,
-        status = hexToDartInt(map['status'] as String) == 1;
+        status = map['status'] != null ? hexToDartInt(map['status'] as String) == 1 : null;
```

Comment:
Does omission implicitly mean `0`/`false` here?
```suggestion
        status = map['status'] != null && hexToDartInt(map['status'] as String) == 1;
```

Reply:
If I'm understanding this correct, than this problem appears with private Instances, not using ``byzantiumBlock``-record. (https://github.com/ethereum/web3.js/issues/1138)

I don't know if it's always safe to assume that it did not pass. It just doesn't report it, or?

Reply:
I would prefer to make `status` nullable as well :+1: But I think we should add some documentation on the `status` field to indicate that it can be null if the node didn't indicate a status.